vault/notebook/git.md

```diff
@@ -0,0 +1,477 @@
+# An Introduction To Git
```

Comment:
I don't think we want this - see comment in spreadsheet.

Reply:
Actioned: https://github.com/life-itself/community/pull/202/commits/796e1c9f27751ba2e28aa3726366e9a4dbba1939